streams/src/test/java/org/apache/kafka/streams/integration/QueryableStateIntegrationTest.java

```diff
@@ -93,6 +93,7 @@
     private static final long WINDOW_SIZE = TimeUnit.MILLISECONDS.convert(2, TimeUnit.DAYS);
     private static final int NUM_PARTITIONS = 2;
     private static final int NUM_REPLICAS = NUM_BROKERS;
+    private static final int MAX_WAIT_MS = Integer.getInteger("kafka.test.max.wait.ms", 60000);
```

Comment:
Making timeouts configurable could be a good idea, but it's better done in a general way in its own PR.